frameworks/moveos-stdlib/sources/module_store.move

```diff
@@ -114,7 +113,10 @@ module moveos_std::module_store {
         let (module_names, module_names_with_init_fn, indices) = move_module::sort_and_verify_modules(&modules, package_id);
 
         if (!exists_package(module_object, package_id)) {
-            object::add_field(module_object, package_id, object::new(Package {}));
+            let package = object::add_object_field_with_id(module_object, package_id, Package {});
+            let package_obj_id = object::id(&package);
```

Comment:
`package_obj_id = object::custom_child_object_id<address, Package>(parent_id, package_id)`

Do not need to store it in a field.